src/taskManager/TaskList.java

```diff
@@ -0,0 +1,64 @@
+package taskManager;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private List<Task> taskList = new ArrayList<>();
+
+    public void addTask(Task task) {
+        taskList.add(task);
+    }
+
+    //Why pass whole Task? mb pass only title
+    public void removeTask(String title) {
+        boolean isRemoved = taskList.removeIf(task -> {
+            return task.getTitle().equals(title);
+        });
+
+        String message = isRemoved ? "Task removed" : "Invalid title input";
+        System.out.println(message);
+
+        //Another implementation
+//        var tasksToRemove = new ArrayList<Task>();
+//        for (Task task: taskList) {
+//            if (task.getTitle().equals(title)) {
+//                tasksToRemove.add(task);
+//            }
+//        }
+//        taskList.removeAll(tasksToRemove);
+
+    }
+
+    public void printTasks() {
+        StringBuilder list = new StringBuilder();
```

Comment:
An alternative could be storing task description in a collection and then using "join" on this collection to build a single concatenated string. In this case it won't have a surplus line separator in the end.

Reply:
Another option is just printing immediately to the console without storing intermediate result